jsk_pcl_ros/CMakeLists.txt

```diff
@@ -176,6 +177,8 @@ jsk_pcl_nodelet(src/cluster_point_indices_decomposer_z_axis_nodelet.cpp
   "jsk_pcl/ClusterPointIndicesDecomposerZAxis" "cluster_point_indices_decomposer_z_axis")
 jsk_pcl_nodelet(src/resize_points_publisher_nodelet.cpp
   "jsk_pcl/ResizePointsPublisher" "resize_points_publisher")
+jsk_pcl_nodelet(src/rearrange_bounding_box_publisher_nodelet.cpp
+  "jsk_pcl/RearrangeBoundingBoxPublisher" "rearrange_bounding_box_publisher")
```

Comment:
> ResizePointsPublisher.

What a bad name..
I don't think you have to follow that name, but if you'd like to follow what about `rearranged_bounding_box_publisher`?